tez-ui/src/main/webapp/bower-shrinkwrap.json

```diff
@@ -26,6 +23,9 @@
   "https://github.com/dockyard/qunit-notifications.git": {
     "0.1.1": "7a13f6dba5a340e1cb9e0b64c1c711e4d7edaca1"
   },
+  "https://github.com/eligrey/FileSaver.js.git": {
+    "1.3.4": "b4a918669accb81f184c610d741a4a8e1306aa27"
```

Comment:
I think it would make sense to point to same commit (https://github.com/eligrey/FileSaver.js/commit/b7cf622909258086bc63ad764d08fcaed780ab42) in this PR so there are no extra changes. 
Upgrading can be handled in separate commit.